src/Components/PostCard/PostCard.jsx

```diff
@@ -22,6 +38,14 @@ const PostCard = ({ eachPost, deletedPost, likePost }) => {
     likePost(_id);
   };
 
+  //Post Edit Handler
+
+  const postEditHandler = ({ caption, content }) => {
+  setEditPostModal(true)
+  setPostEditData({content: content, caption: caption, id: _id })
```

Comment:
({content: content, caption: caption, id: _id }) can be written as ({ content, caption, id: _id  }) . It's a shortcut if the key-value pair are same